superbench/common/utils/topo_aware.py

```diff
@@ -31,6 +34,32 @@ def search(self, pattern, string, flags=0):
         return self.matched
 
 
+def gen_ibstat_file(ibstat_file):
+    """Generate ibstat file for each node with specified path.
+
+    Args:
+        ibstat_file (str): path of ibstat output.
+    """
+    comm = MPI.COMM_WORLD
+    name = MPI.Get_processor_name()
```

Comment:
Is it guaranteed to be hostname?

Reply:
I think so, if the node num is lagrer than 1 under mpi mode.
https://github.com/microsoft/superbenchmark/blob/b5c7c85d178cf4914bec4526bebd55a962f74de4/superbench/runner/runner.py#L151